lib/typesense/helper.rb

```diff
@@ -34,6 +34,15 @@ def create
           type: 'geopoint',
           facet: false,
           optional: true
+        }, {
+          name: 'name',
+          type: 'string',
+          sort: true
+        },{
+          name: 'name_facet',
```

Comment:
I'm going to walk back my previous note, I don't think we'll need `name_facet` to be sortable.